scripts/ensure_database_versions.py

```
@@ -58,8 +60,21 @@ def main() -> int:
+    # Use GITHUB_WORKSPACE in CI, otherwise use script's parent directory
+    if "GITHUB_WORKSPACE" in os.environ:
+        repo_root = Path(os.environ["GITHUB_WORKSPACE"])
+    else:
+        repo_root = Path(__file__).resolve().parents[1]
+
+    try:
+        ensure_versions_file(repo_root / "cache" / "food_db" / "database_versions.json")
+    except OSError:
+        # Let OSError and its subclasses propagate for callers/tests that expect it.
+        raise
+    except Exception as exc:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)